src/App.tsx

```diff
@@ -92,14 +80,9 @@ export const App = () => {
 
           <div className={styles.content}>
             <ChatView
-              introText={modelConfig.introText}
-              cautionText={defaultCautionText}
-              messages={messages}
-              onSubmit={handleChatCompletion}
```

Comment:
All of these values can be pulled directly from context or model config, so no need to drill them.